actionpack/lib/action_controller/metal/content_security_policy.rb

```diff
@@ -28,6 +51,18 @@ def content_security_policy(enabled = true, **options, &block)
         end
       end
 
+      # Allows overriding the global `content_security_policy_report_only`
```

Comment:
Not sure. I started renaming all occurences of "Content Security Policy" to "Content-Security-Policy" but I don't think that is correct. The headers are called "Content-Security-Policy" and "Content-Security-Policy-Report-Only". But I think mentioning the headers might be too much detail for the controller overrride documentation.
I think I'd rather change "Content-Security-Policy" back to  "Content Security Policy".

Reply:
> I've changed the phrasing slightly for Content-Security-Policy-Report-Only,
> because I think you can only override all of it, not just parts.

Yes, thank you!  (I was swapping between the Content-Security-Policy and Content-Security-Policy-Report-Only docs when I wrote that.)